### ISSUE TITLE
Add a bunch of templates and guides

### ISSUE THREAD

**Issue by wgyt** [wgytcraft/hosting#35]
**Is your feature request related to a problem? Please describe.**
Make it possible for wgytcraft to be used better
**Describe the solution you'd like**
Add a bunch of templates and guides
**Describe alternatives you've considered**
none
**Additional context**
pls

**Comment by wgyt** [comment]
# Ideas
cc @retronbv and @GrahamSH-LLK 
- [ ] Blog website
- [ ] Documentation website (like documentation.wgyt.tk)
- [ ] other ideas would be cool

**Comment by retronbv** [comment]
no blog dumbo

**Comment by wgyt** [comment]
> no blog dumbo

why @retronbv

**Comment by retronbv** [comment]
because blog estupido

**Comment by wgyt** [comment]
> because blog estupido

I don't agree with this, it could be a website made with markdown or something, not just a blog

**Comment by retronbv** [comment]
Why do we need a blog?

**Comment by wgyt** [comment]
> Why do we need a blog?

these are template ***ideas*** thats why this issue is called "Add a bunch of templates and guides"

**Comment by thecoder876** [comment]
@wgyt can i help with docs pls?

**Comment by wgyt** [comment]
> @wgyt can i help with docs pls?

sure, https://github.com/wgytcraftf/help

**Comment by thecoder876** [comment]
> > [mention removed] can i help with docs pls?
> 
> sure, https://github.com/wgytcraftf/help

@wgyt ok

**Comment by wgyt** [comment]
Now that we merged #79 I think this would be much easier to do.

cc @retronbv @GrahamSH-LLK @GrahamSH-LLK

**Comment by retronbv** [comment]
@wgyt why ping graham twice lmao

**Comment by wgyt** [comment]
> @wgyt why ping graham twice lmao

I meant to ping @thecoder876 lol

**Comment by retronbv** [comment]
> > @wgyt why ping graham twice lmao
> 
> I meant to ping @thecoder876 lol

Lol

**Comment by thecoder876** [comment]
@wgyt Imao now i was pinged twice
